LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2021 University Corporation for Atmospheric Research
```

Comment:
We could probably update this to be 2024

Reply:
I thought about that... But the contents of the license aren't changing, so I figured the date should stay the same?